### ISSUE TITLE
Allow ant style patterns to match files to render

### ISSUE THREAD

**Issue by AlexanderZobkov** [asciidoctor/asciidoctor-maven-plugin#174]
Currently it's possible to configure asciidoctor-maven-plugin to render:
- (1) one specific file by setting *sourceDocumentName* attribute or 
- (2) all files in one specific directory and its subdirectories by setting *sourceDirectory* attribute

In case it's needed to render multiple documents with one Maven execution (for maintenability reason) and specified directory contains fragments or directories with fragments for two or more big documents we actually want to render, it will results in rendering of all files (all fragments and big documents will be rendered into a separate pdf files, for example).

It would be great to allow selecting documents to be rendered by using ant style patterns.

**Comment by mojavelinux** [comment]
Agreed. We should align with the Gradle plugin on this one. See https://github.com/asciidoctor/asciidoctor-gradle-plugin#usage (sources)

**Comment by abelsromero** [comment]
Hi @AlexanderZobkov, the last PR should fix your issue. I'd like to confirm with you that it helps before closing it.

The details of the preview are here:
https://bintray.com/abelsromero/maven/asciidoctor-maven-plugin/1.5.4-preview.2/

You can create a project using this version and test it.

```
<sourceDocumentName>./README.adoc</sourceDocumentName>
<resources>
    <resource>
        <directory>${generated_path}</directory> <!-- you can just use . also -->
        <exclude>**/.svn</exclude> <!-- does not copy anything -->
    </resource>
</resources>
```
